x-pack/plugins/triggers_actions_ui/public/application/sections/rule_form/rule_add.test.tsx

```diff
@@ -214,8 +213,8 @@ describe('rule_add', () => {
     );
 
     expect(wrapper.find('input#ruleName').props().value).toBe('Simple status rule');
-    expect(wrapper.find('[data-test-subj="tagsComboBox"]').first().text()).toBe('uptimelogs');
     expect(wrapper.find('[data-test-subj="intervalInput"]').first().props().value).toEqual(1);
+    expect(wrapper.find('[data-test-subj="tagsComboBox"]').first().text()).toBe('uptimelogs');
```

Comment:
guess this file might contain some leftovers from recent changes that could be cleaned up since not relevant for this PR ?